pkg/constants/constants.go

```diff
@@ -220,6 +221,31 @@ var (
 	}
 )
 
+var (
+	//RawIngressNamePrefix ...
+	RawIngressNamePrefix = KFServingName + "-ingress"
+)
+
+// raw k8s deployment, resource exist check result
+type CheckResultType int
+
+const (
+	CheckResultCreate  CheckResultType = 0
+	CheckResultUpdate  CheckResultType = 1
+	CheckResultExisted CheckResultType = 2
+	CheckResultUnknown CheckResultType = 3
+)
+
+// GetRawServiceLabel generate native service label
+func GetRawServiceLabel(service string, namespace string) string {
+	return "isvc." + namespace + "." + service
+}
+
+// GetRawServiceIngressName generate native service ingress name
+func GetRawServiceIngressName(service string, namespace string) string {
+	return RawIngressNamePrefix + "-" + namespace + "-" + service
```

Comment:
I think we probably need a `domainTemplate` like in knative, the convention is `service.namespace` or `server-namespace`.